src/components/Content/Content.tsx

```diff
@@ -32,10 +35,10 @@ const Content: React.FC = () => {
         </ul>
       </aside>
       <main className={styles.contentMain}>
-        {loading ? (
+        {props.loading ? (
```

Comment:
Buraya error ve no result için de basit birer view ekleyelim. Loading, no result, error, düzgün veri listeleme şeklinde